webapp/channels/src/components/widgets/modals/components/checkbox_setting_item.tsx

```diff
@@ -37,22 +37,26 @@ export default function CheckboxSettingItem({
     descriptionAboveContent = false,
 }: Props) {
     const content = (
-        <fieldset
+        <div
             key={inputFieldData.name}
             className='mm-modal-generic-section-item__fieldset-checkbox-ctr'
```

Comment:
nit: maybe change the name of this class, since it no longer is a fieldset but a div.